src/core/CoolBoard.cpp

```diff
@@ -507,21 +507,28 @@ void CoolBoard::printMqttState(int state) {
 }
 
 void CoolBoard::mqttConnect() {
-  uint8_t i = 0;
+  uint8_t i = mqttRetries;
+  uint8_t max = mqttRetries + 5;
```

Comment:
No, you shall not delete a `#define`'d constant and replace it by a litteral "magic number" :) !
Keep that `#define MQTT_RETRIES 5`, keep `i = 0`, do not mix the two variables, as it makes your intent unclear and the code overly complicated.